core/server/common/src/main/java/alluxio/StorageTierAssoc.java

```diff
@@ -21,7 +19,7 @@
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Creates a two-way mapping between {@link StorageTier} aliases and ordinal numbers from the given
+ * Creates a two-way mapping between storage tier aliases and ordinal numbers from the given
```

Comment:
yes, this class is in `core-server-common` and `StorageTier` is in `core-server-worker`